Source/MLX/Ops.swift

```diff
@@ -1650,8 +1716,8 @@ public func sinh(_ array: MLXArray, stream: StreamOrDevice = .default) -> MLXArr
 ///
 /// ### See Also
 /// - <doc:arithmetic>
-/// - ``softMax(_:axis:stream:)``
-/// - ``softMax(_:stream:)``
+/// - ``softMax(_:axis:precise:stream:)``
+/// - ``softMax(_:precise:stream:)``
```

Comment:
When doing the doc build I noticed a couple of these references were out of date (we added `precise:` in the last drop).